lib/config-validator.ts

```diff
@@ -134,7 +140,7 @@ type PackageJson = {
         const file = process.env.RENOVATE_CONFIG_FILE ?? 'config.js';
         logger.info(`Validating ${file}`);
         try {
-          await validate(file, fileConfig, strict);
+          await validate(file, fileConfig, strict, true);
```

Comment:
This function takes 5 params, you have provided 4. You are setting isPreset=true when you presumably mean isGlobalConfig=true.

This is a lesson to avoid optional params like this! Please refactor so they are mandatory and passed each time